test/serializers/default_include_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    class DefaultIncludeTest < ActiveSupport::TestCase
+      class Blog < ActiveModelSerializers::Model
+        attr_accessor :id, :name, :posts
+      end
+      class Post < ActiveModelSerializers::Model
+        attr_accessor :id, :title, :author
+      end
+      class Author < ActiveModelSerializers::Model
+        attr_accessor :id, :name
+      end
+      class BlogSerializer < ActiveModel::Serializer
+        attributes :id
+        attribute :name, key: :title
+
+        has_many :posts
+      end
+      class PostSerializer < ActiveModel::Serializer
+        default_include [:author]
+        attributes :id, :title
+
+        has_one :author
```

Comment:
I can see the advantage of the `fields` option, but wouldn't be easy to confuse `include` and `exclude` with `only` and `except`? Actually, aren't they doing the same thing?